packages/reporting-server/README.md

```diff
@@ -39,7 +36,7 @@ RMF_REPORT_REST_SERVER_CONFIG='my_config.py' reporting_server
 * MySQL
 * MariaDB
 
-by default it uses a in-memory sqlite instance, to use other databases, install rmf-server with the relevalent extras
+by default it uses a PostgreSQL instance, to use other databases, install rmf-server with the relevalent extras
```

Comment:
```suggestion
by default it uses a PostgreSQL instance, to use other databases, install reporting-server with the relevalent extras
```

Whats the reason to move from sqlite by default to postgres?

Reply:
We can use the persistent SQLite as well, I have no problems at all. 

Let's suppose I want to try a GET request to get data, every time that the system reloads for any change, all the data is wiped out (because it's a DB on memory). So you need to run either a post to fill the DB or a script. With the default Tortoise's migration system, all the tables are created automatically on a fresh DB (every time we reloaded). With aerich, it's different because you need to run a command for the tables to be created, so that's why an in-memory wasn't a good option. Also with the default Tortoise's migration system, I had a lot of problems updating the tables with new fields, and I do not have it anymore with aerich (but I need to create a migration and apply it to the database).